src/components/chips/js/chipsDirective.js

```diff
@@ -1,269 +1,273 @@
-angular
-    .module('material.components.chips')
-    .directive('mdChips', MdChips);
+(function(){
```

Comment:
Github's diff is making it really hard to see what's actually changed in this file.